src/main/java/bio/terra/service/dataset/DatasetJsonConversion.java

```diff
@@ -104,7 +104,8 @@ public static DatasetModel populateDatasetModelFromDataset(
             .phsId(dataset.getPhsId())
             .selfHosted(dataset.isSelfHosted())
             .predictableFileIds(dataset.hasPredictableFileIds())
-            .tags(dataset.getTags());
+            .tags(dataset.getTags())
+            .resourceLocks(dataset.getResourceLocks());
```

Comment:
TDR doesn't seem to do this, but I wonder why `populateDatasetModelFromDataset()` isn't a method on `Dataset` (e.g. `Dataset.toModel()`). My usual rule of thumb is, if you have a static method that accepts an object as its first parameter (and you own that object) then it's likely that it should be a method on that object. There's an intellij refactor which can move the method and make it an instance method.

Reply:
Ticketed separately https://broadworkbench.atlassian.net/browse/DR-3109